extensions/blocks/image-compare/image-compare.php

```diff
@@ -49,12 +49,21 @@ function load_assets( $attr, $content ) {
 /**
  * Render image compare block for AMP
  *
- * @param array $attr Array containing the image-compare block attributes.
+ * @param array  $attr    Array containing the image-compare block attributes.
+ * @param string $content String containing the image-compare block content.
  *
  * @return string
  */
-function render_amp( $attr ) {
-	// @TODO: what to show for AMP view?
-	// :shrug-emoji:
-	return '';
+function render_amp( $attr, $content ) {
+	// Attributes that are parsed in JavaScript are not passed in, so to get
+	// the URLs we need to parse the content in PHP, which is rather lame.
```

Comment:
What if we were to not provide any attribute source when declaring the block attributes? It would allow us to get those attributes here in PHP.

Reply:
This is counter to the intent of blocks, the goal is to parse all the data from markup itself. Without a source and storing it in the comment field duplicates in the attributes and the markup.

Reply:
Switched to remove source. This can also make it more flexible in the future if we need to change the markup.